modules/ROOT/pages/getting-started.adoc

```diff
@@ -6,6 +6,10 @@
 
 There are three Fedora CoreOS (FCOS) update streams available: `stable`, `testing`, and `next`. In general, you will want to use `stable`, but it is recommended to run some machines on `testing` and `next` as well.
 
+Each stream has a canonical URL representing its current state in JSON format.  For example, the URL for the stable stream is: https://builds.coreos.fedoraproject.org/streams/stable.json
+While Fedora CoreOS does automatic in-place updates, it is generally a good practice to start provisioning new machines from the latest "bootimages".
```

Comment:
The term `bootimage` is not used in FCOS, since (as this points out) everyone should always start from the latest release, so let's not introduce the term here.

Reply:
Do you have an alternative suggestion?  Just ..."from the latest images"?

Reply:
Updated to use just "images".